src/pages/NotFoundPage/NotFoundPage.tsx

```diff
@@ -1,10 +1,35 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
+
 import styles from './NotFoundPage.module.scss';
 
 export const NotFoundPage: React.FC = () => {
+
   return (
     <div className={styles.page}>
-      <h1>Not Found Page</h1>
+      <div className={styles.error}>
+        <div className={styles.number}>4</div>
+        <div className={styles.illustration}>
+          <div className={styles.circle}></div>
+          <div className={styles.clip}>
+            <div className={styles.paper}>
+              <div className={styles.face}>
+                <div className={styles.eyes}>
+                  <div className={`${styles.eye} ${styles.eye_left}`}></div>
```

Comment:
```suggestion
import cn from 'classnames'
                  <div className={cn(styles.eye, styles.eye_left)}></div>
```

Просто давай так. Не через строку а через classnames